### DISCUSSION TITLE
How to re-detect viable collectors?

### DISCUSSION THREAD

**Discussion by shalak** [netdata/netdata#14380] [Q&A] [answered]
I installed netdata, and then, some time after that, I installed postfix, among other tools.

How can I make netdata re-detect all the viable collectors, so now it covers postfix as well?

I tried re-running the one-liner install script, but it didn't add any new collectors (but when I installed netdata for the first time, on different server that already had postfix installed, it shows collector for it).

**Comment by andrewm4894** [comment, marked as answer]
@shalak i think best way would be to [restart the agent](https://learn.netdata.cloud/docs/configure/start-stop-restart)

Not sure if there is a way to do this via the [netdatacli](https://learn.netdata.cloud/docs/agent/cli) since the agent needs to spawn more stuff when it comes to collectors is my understanding.

**Comment by shalak** [comment]
> @shalak i think best way would be to [restart the agent](https://learn.netdata.cloud/docs/configure/start-stop-restart)

Haha! So it's simple as that? 😅 I didn't even notice that the reinstall did not restart the service! :) Now the postfix showed up! Thanks!
